models/reviews.model.js

```diff
@@ -37,33 +37,54 @@ exports.updateReviewById = (id, inc_votes) => {
     });
 };
 
-exports.selectReviews = (query) => {
+exports.selectReviews = (category, sort_by = "created_at", order = "desc") => {
   const validCategories = [
```

Comment:
I dont think this way of handling the categories will work in the long run, we had to use the process of green listing for sort and order because they are not row values and parameterised queries won't work, but we can use parameterised queries for categories